templates/download/desktop/index.html

```diff
@@ -78,37 +78,37 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
                 <a class="p-tabs__link"
                    role="tab"
                    aria-selected="true"
-                   id="release-notes-tab"
-                   href="#release-notes"
-                   aria-controls="release-notes">What's new</a>
+                   id="release-notes-tab-{{ releases.lts.slug }}"
+                   href="#release-notes-{{ releases.lts.slug }}"
+                   aria-controls="release-notes-{{ releases.lts.slug }}">What's new</a>
               </li>
               <li class="p-tabs__item">
                 <a class="p-tabs__link"
                    role="tab"
                    aria-selected="false"
-                   id="system-requirements-tab"
-                   href="#system-requirements"
-                   aria-controls="system-requirements"
+                   id="system-requirements-tab-{{ releases.lts.slug }}"
+                   href="#system-requirements-{{ releases.lts.slug }}"
+                   aria-controls="system-requirements-{{ releases.lts.slug }}"
                    tabindex="-1">System requirements</a>
               </li>
               <li class="p-tabs__item">
                 <a class="p-tabs__link"
                    role="tab"
                    aria-selected="false"
-                   id="how-to-install-tab"
-                   href="#how-to-install"
-                   aria-controls="how-to-install"
+                   id="how-to-install-tab-{{ releases.lts.slug }}"
+                   href="#how-to-install-{{ releases.lts.slug }}"
+                   aria-controls="how-to-install-{{ releases.lts.slug }}"
                    tabindex="-1">How to install</a>
               </li>
             </ul>
           </nav>
         </div>
         <!-- End tab container -->
         <!-- tab 1 -->
-        <div id="release-notes"
+        <div id="release-notes-{{ releases.lts.slug }}"
              class="p-tabs__content"
              role="tabpanel"
-             aria-labelledby="release-notes-tab">
+             aria-labelledby="release-notes-tab-{{ releases.lts.slug }}">
           <ul class="p-list--divided">
             <li class="p-list__item is-ticked">New Desktop installer with support for autoinstall</li>
             <li class="p-list__item is-ticked">New App Center and Firmware Updater applications</li>
```

Comment:
Can't edit the line directly, but there's been a change in the doc that affects line 122. The text in the brackets is supposed to add context for screen readers, but it shouldn't be shown visually. Example implementation [here](https://github.com/canonical/ubuntu.com/blob/3774902ecc0c2d9356fcf8d68ee6fb16485f91a7/templates/base_index.html#L453) and screenshot since I can't link the comment for some reason: 
![image](https://github.com/user-attachments/assets/2b061bc9-0258-48e1-841e-133b25726bd9)